scripts/mytop.sh

```diff
@@ -315,6 +328,22 @@ foreach (@variables)
     }
 }
 
+my ($has_is_processlist, $has_time_ms, $has_progress);
+$has_is_processlist = $has_time_ms = $has_progress = 0;
+
+## Check if the server has the INFORMATION_SCHEMA.PROCESSLIST table for backward compatibility
```

Comment:
L: 334-345 are going out of 80 char rule, we are trying to stick to as much as possible to 80 chars long line of code (including comments). The same is for other part of your code L:812.